pocs/utils/__init__.py

```diff
@@ -48,6 +48,42 @@ def flatten_time(t):
     return t.isot.replace('-', '').replace(':', '').split('.')[0]
 
 
+# This is a streamlined variant of PySerial's serialutil.Timeout.
+class Timeout(object):
+    """Simple timer object for tracking whether a time duration has elapsed.
+
+    Attribute `is_non_blocking` is true IFF the duration is zero.
```

Comment:
I don't understand where non-blocking case. Where would you be using a timeout=0?

Reply:
Yes, timeout==0 means that it will always be considered to have expired.